style.css

```
@@ -0,0 +1,179 @@
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:wght@300&family=Roboto&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body, button {
+    font-family: Roboto, sans-serif;
+}
+
+.box {
+    flex-grow: 1;
+    position: relative;
+}
+
+p {
+    color: #0C0C0C;
+}
+h1,h2,h3 {
+    font-family: Merriweather, serif;
+}
+
+h1, h2 {
+    text-align: center;
+    color: white;
+}
+
+h1 {
+    flex-direction: column;
+    font-size: xx-large;
+}
+
+h2 {
+    color: #4845E4;
+    font-size:x-large;
+}
+
+.header {
+    background-color: #4845E4;
+    padding-block: 150px;
+    /*padding-bottom:150px;
+    padding-top: 150px;*/
+    flex-direction: column;
+    position: relative;
+}
```

Hier moet je nog een display zetten:
```suggestion
    display: flex;
}
```